ingenialink/dictionary.py

```diff
@@ -305,6 +314,15 @@ def _read_xdf_register(self, register):
                 raise exc.ILAccessError(f'The access type {access_aux} does not exist for the register: '
                                         f'{current_read_register[self.AttrRegDict.IDENTIFIER]}')
 
+            # Address type
+            address_type_aux = register.attrib['address_type']
+
+            if address_type_aux in self.address_type_xdf_options:
+                current_read_register[self.AttrRegDict.ADDRESS_TYPE] = self.address_type_xdf_options[address_type_aux]
+            else:
+                raise exc.ILAccessError(f'The address type {address_type_aux} does not exist for the register: '
```

Comment:
[This ](https://github.com/ingeniamc/ingenialink-python/blob/master/ingenialink/exceptions.py#L132)exception looks better suited for this case. What do you think?

Reply:
The exception is not documented.

Reply:
> [This ](https://github.com/ingeniamc/ingenialink-python/blob/master/ingenialink/exceptions.py#L132)exception looks better suited for this case. What do you think?

I don't know, it's not a register value. Should we create a new exception for this case?